src/lib/Zuora.js

```diff
@@ -91,7 +91,7 @@ export class Zuora {
         // TODO SEE HOW TO DETECT FAILURES OR ANY OTHER SPECIAL CASE HERE
           let fileData = {
             batchName: batch.name,
-            fileName: `${batch.name}_${deliveryDate}.csv`,
+            fileName: `${batch.name}_${deliveryDate}_${batch.fileId}.csv`,
```

Comment:
Will this file renaming break @jacobwinch 's Home Delivery fulfilment lookup?

Reply:
Currently I'm only interested in the Salesforce filename, since my lookup only cares about the data that the distributor _actually_ used. However, once we swap over we'll need a way of identifying the file which the distributor picked up, although I suspect that's beyond the scope of Alex's PR.

Reply:
This is just the stored values from Zuora. If a fulfilment is rerun then currently the Zuora output is lost.